pkg/model/iam/iam_builder.go

```diff
@@ -293,11 +293,13 @@ func addRoute53ListHostedZonesPermission(p *IAMPolicy) {
 }
 
 // IAMPrefix returns the prefix for AWS ARNs in the current region, for use with IAM
-// it is arn:aws everywhere but in cn-north, where it is arn:aws-cn
+// it is arn:aws everywhere but in cn-north and us-gov-west-1
```

Comment:
I would put a new document into kops/docs

Up to you about tickets. If they are different parts to the system, I would probably use different tickets.